### PR TITLE
Move track beacons guide under Analytics and upgraded to latest sdk APIs

### PR DESCRIPTION
**Pull request by emdobrin**
## Description

- Moved the track beacon user guide under Analytics
- Updated samples to use latest version of the APIs, Android SDK 2.x and iOS 4.x

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -744,10 +748,6 @@ module.exports = {
                     {
                         title: "iOS App Extensions",
                         path: "documentation/user-guides/app-extension"
-                    },
-                    {
-                        title: "Tracking beacons",
-                        path: "documentation/user-guides/track-beacon"
```
Comment (emdobrin) [inline gatsby-config.js:750]: @calebk1 @swarna04 this change breaks the old link, I think we should be good for these small instances but let me know it you consider we should use redirects

**Review by cacheung — APPROVED** [review]
Looks good!

File: gatsby-config.js (hunk 1 above)
Comment (emdobrin) [reply gatsby-config.js:750]: Following up on this one - decided we will create redirects for all these links once all the changes are merged into the resources-and-guides branch

File: src/pages/documentation/adobe-analytics/track-beacon.md [hunk 2]
```diff
@@ -7,7 +7,9 @@ import ClearBeaconIos from './tabs/track-beacon/clear-beacon/ios.md'
 
 ## Emulate the trackBeacon call from the v4 SDKs
 
-The `trackBeacon` call is no longer available in the Experience Platform SDKs. Now, you need to manually send beacon tracking data to your Adobe Analytics server and create the rules based on a user's proximity to your beacons. This solution relies on the [Profile extension](../profile/index.md) to generate beacon-related rules.
+Sending beacon data to Adobe Analytics for reporting can be an effective to understand how your audience can interact with physical landmarks.
+
+The `trackBeacon` API from v4 SDKs is no longer available in the Experience Platform SDKs. Now, you need to manually send beacon tracking data to your Adobe Analytics server and create the rules based on a user's proximity to your beacons. This solution relies on the [Profile extension](../profile/index.md) to generate beacon-related rules.
```
[the remark this answers is not in the document]
Comment (emdobrin) [reply src/pages/documentation/adobe-analytics/track-beacon.md:12]: I think this rephrasing is a bit too simplified and looses some of the context, the meaning is that compared to v4 there is no dedicated API you can send beacons using these two extensions and rules.

**Review by calebk1 — APPROVED** [review]
LGTM!

**Pull request merged**